spec/unit/pops/benchmark_spec.rb

```diff
@@ -85,15 +67,15 @@ def json_load(string)
 
   it "marshal1", :profile => true do
     parser = Puppet::Pops::Parser::EvaluatingParser.new()
-    model = parser.parse_string(code).current
+    model = parser.parse_string(code).model
     dumped = Marshal.dump(model)
     m = Benchmark.measure { 10000.times { Marshal.load(dumped) }}
     puts "Marshal1: #{m}"
   end
 
   it "rgenjson", :profile => true do
     parser = Puppet::Pops::Parser::EvaluatingParser.new()
-    model = parser.parse_string(code).current
+    model = parser.parse_string(code).model
     dumped = json_dump(model)
     m = Benchmark.measure { 10000.times { json_load(dumped) }}
     puts "RGen Json: #{m}"
```

Comment:
This mentions RGen - should say Pcore as the test now benchmarks the pcore serialization in comparison with a Ruby Marchal. (i.e. also change the `it "rgenjson"` to `it "pcore_json". Could also add a pcore msgpack variant here for completeness.